VideoLocker/src/main/java/org/edx/mobile/player/Player.java

```diff
@@ -76,6 +75,7 @@ private void init() {
         lastCurrentPosition = 0;
         lastDuration = 0;
         isFrozen = false;
+        this.autoHideControls = autoHideControls;
```

Comment:
Why not just default this to false to be consistent with the other variables. 

Also, I don't think `this` is needed.